sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -476,7 +476,7 @@ def __init__(self, label='ToList'):
     super(ToList, self).__init__(label)
 
   def expand(self, pcoll):
-    return pcoll | core.CombineGlobally(self.label, ToListCombineFn())
+    return pcoll | self.label >> core.CombineGlobally(ToListCombineFn())
```

Comment:
I think we should remove these `self.label` things. (Not necessarily in this PR, but definitely a backward incompatible change.)

How would this work in for example
`label1 >> ToList(label2)`